packages/analytics/src/Analytics.ts

```diff
@@ -142,10 +179,10 @@ class Analytics {
   /**
    * Sets the consent and passes it to the registered integrations.
    *
-   * @param {object} data - Consent object to be stored.
-   * @returns {Promise<Analytics>} Promise that will resolve with the instance that was used when calling this method to allow chaining.
+   * @param data - Consent object to be stored.
```

Comment:
Nitpick: add an empty line between the @params and the @return. valid for all cases in this document

Reply:
Done! Thanks!